definitions/contested/json/ComplexTypes/ComplexTypes.json

```diff
@@ -804,7 +822,7 @@
     "ID": "FR_ct_directionDetailCollection",
     "ListElementCode": "localCourt",
     "FieldType": "FR_ct_courtList",
-    "ElementLabel": "This would usually be the applicants local Court",
```

Comment:
this shouldn't be remove. This is standard message

Reply:
This has been added here as part of FR_ct_courtList
https://github.com/hmcts/finrem-ccd-definitions/pull/1040/files#diff-79dfc01a44465d8eee8ca502c2d37b3032b01323dec1db40713e022e1e57dbb2R350

{
    "LiveFrom": "01/01/2017",
    "ID": "FR_ct_courtList",
    "ListElementCode": "localCourtLbl",
    "FieldType": "Label",
    "ElementLabel": "This would usually be the applicants local Court",
    "SecurityClassification": "Public"
  },